src/xFactor/Person.java

```diff
@@ -0,0 +1,122 @@
+package xFactor;
+
+public class Person {
+	private String firstName;
+	private String lastName;
+	private int age;
+	private String gender;
+	private String birthPlace;

```

Comment:
you can name this homeTown or just city, birthPlace does not sound right. This also can different class - town